src/sentry/integrations/mixins/issues.py

```diff
@@ -99,17 +100,26 @@ def get_performance_issue_description_data(self, event):
         an integration's ticket description. Each integration will need to take
         this data and format it appropriately.
         """
-        spans, matched_problem = get_span_and_problem(event)
-        if not matched_problem:
-            return ""
+        if isinstance(event, GroupEvent) and event.occurrence is not None:
```

Comment:
not sure if this is the best way to check whether a group is backed by an occurrence

Reply:
maybe I should use options here